xblocks_contrib/lti/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-Init for the LTIBlock.
+Learning Tools Interoperability (LTI) module.
```

Comment:
```suggestion
THIS BLOCK IS DEPRECATED IN FAVOR OF https://github.com/openedx/xblock-lti-consumer

Learning Tools Interoperability (LTI) module.
```
Just to make it really obvious that this is deprecated